lib/generators/shopify_app/install/install_generator.rb

```diff
@@ -61,6 +61,10 @@ def mount_engine
       def embedded_app?
         options['embedded'] == 'true'
       end
+
+      def refined(array)
+        array.join(' ').tr('"', '')
+      end
```

Comment:
I find this name a bit unclear - can you be explicit and call it `join_array_argument` or `format_array_argument`